packages/components/src/Popover/Popover.css

```diff
@@ -0,0 +1,83 @@
+:root {
+  --card--background-color: var(--color-white);
+  --card--base-padding: var(--space-base);
+  --popover--padding: var(--space-base);
+  --popover--width: calc(var(--base-unit) * 21.875);
+  --dismiss--button-padding: calc(var(--base-unit) / 4);
+}
+
+.header {
+  display: flex;
+  position: "relative";
+  padding: var(--popover--padding);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.dismissButton {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: var(--dismiss--button-padding);
+}
+
+.popover {
+  z-index: 1;
+  width: 100%;
+  max-width: var(--popover--width);
+  border: 1px solid var(--color-grey--light);
```

Comment:
We have some border custom properties here we can use. Need to make them more obvious through docs.

https://github.com/GetJobber/atlantis/blob/master/packages/design/src/borders.css